clients/src/main/java/org/apache/kafka/common/protocol/types/Type.java

```diff
@@ -331,14 +331,14 @@ public String documentation() {
     public static final DocumentedType UUID = new DocumentedType() {
         @Override
         public void write(ByteBuffer buffer, Object o) {
-            final java.util.UUID uuid = (java.util.UUID) o;
+            final org.apache.kafka.common.UUID uuid = (org.apache.kafka.common.UUID) o;
```

Comment:
We have already imported UUID, do we need to qualify here? I can see it was this way before, but not sure why.

Reply:
My best guess is that since the type is also called UUID, originally the qualification was to differentiate. But it doesn't seem to be needed, so I'll update this to just say UUID.